pkg/sink/kafka/sarama.go

```diff
@@ -255,3 +256,40 @@ func getKafkaVersionFromBroker(config *sarama.Config, requestVersion int16, addr
 	}
 	return KafkaVersion, nil
 }
+
+// keepConnAlive is used to keep the connection alive.
+func keepConnAlive(client sarama.Client, done chan struct{}) {
```

Comment:
This method looks good to me, but should be controlled by the context, instead of the done channel.